packages/route-viewer-overlay/src/index.js

```diff
@@ -53,19 +53,17 @@ class RouteViewerOverlay extends MapLayer {
   // TODO: determine why the default MapLayer componentWillUnmount() method throws an error
   componentWillUnmount() {}
 
-  componentDidUpdate(prevProps) {
-    // if pattern geometry just finished populating, update the map points
-    if (
-      !isGeomComplete(prevProps.routeData) &&
```

Comment:
Why was this condition removed?

Reply:
Perhaps the condition needs to include checking whether the routeData has changed too.

Reply:
Other than `path` which I don't think we need to expect changing after render all props that can change should trigger a re-center